### ISSUE TITLE
Remove lodash full dependency?

### ISSUE THREAD

**Issue by heff** [balderdashy/merge-defaults#7]
We've been adding [lodash-modularized](https://www.npmjs.com/browse/keyword/lodash-modularized) to [video.js](https://github.com/videojs/video.js), and it'd be nice to use this function like those, without pulling in all of lodash.

Also, you're using `_.isArray(arguments[0]) || _.isDate(arguments[0]`, to exclude properties from recursion, but could you just use `_.isPlainObject` instead? Do you need to recursively merge non-plain objects?

Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @heff.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
